code/modules/mob/living/carbon/human/species/necromorph/mob.dm

```diff
@@ -0,0 +1,35 @@
+/*
+	Code for the necromorph mob.
+	Most of this is a temporary hack because we don't have proper icons for parts.
+
+`	I am well aware this is not how human mobs and species are supposed to be used
+*/
+/mob/living/carbon/human/necromorph
```

Comment:
I'm really unsure about having necros be humans...
Do they need to be able to pick up things? Would simplemobs not make this much easier?

Reply:
This is settled, we spent three days mulling it over collectively. Human subtype is needed in the future for dismemberment